tests/scan/test_basic.py

```diff
@@ -806,8 +806,8 @@ def get_sum_of_grad(input0, input1):
         # Call verify_grad to ensure the correctness of the second gradients
         floatX = config.floatX
         inputs_test_values = [
-            np.random.random((2, 3)).astype(floatX),
-            np.random.random(3).astype(floatX),
+            np.random.default_rng(utt.fetch_seed()).random((2, 3)).astype(floatX),
+            np.random.default_rng(utt.fetch_seed()).random(3).astype(floatX),
```

Comment:
It looks like this will reset the state.  Try creating `rng = np.random.default_rng(utt.fetch_seed())` once and using `rng` for both these cases.

Reply:
Already tried, and also tried passing that rng all the way to `verify_grad` it still fails. And strangely I can't replicate any of this locally. I have to commit and 'uncommit' each time.

Reply:
Well, let's keep that change anyway. 

Otherwise, the failures could simply be due to the underlying RNG change generating a new set of samples, so we can update the test for this difference (e.g. the accepted error ranges and such), find a more advantageous seed, or something else.  Changing the seed is the least preferrable, though.